src/main/resources/twilio-go/api.mustache

```diff
@@ -39,32 +39,39 @@ type {{{nickname}}}Params struct {
 {{/hasOptionalParams}}
 
 // {{operationId}}{{#summary}} {{{.}}}{{/summary}}{{^summary}} Method for {{operationId}}{{/summary}}
-//
 {{#notes}}
-// {{notes}}
 //
+// * {{notes}}
```

Comment:
I thought this was all changed recently?

Reply:
It was but the formatting was a little weird in some cases where there was no path param. I can work with @thinkingserious to get these changes merged or move this to a new PR.

Reply:
It was but the formatting was a little weird in some cases where there was no path param. I can work with @thinkingserious to get these changes merged or move this to a new PR.